src/deploy/functions/checkIam.ts

```diff
@@ -104,3 +118,103 @@ export async function checkHttpIam(
   }
   logger.debug("[functions] found setIamPolicy permission, proceeding with deploy");
 }
+
+/** Callback function to find all v2 service types in from events */
+function reduceEventsToServicesV2(filtered: Set<Service>, option: backend.Endpoint) {
```

Comment:
It was called "option" since this is a callback for a reduce function and we have the option to take each endpoint, I've changed it to "endpoint" in the latest commit